Feature/Skills.feature

```diff
@@ -0,0 +1,23 @@
+﻿Feature: Skill
+User wants to add Skills into Profile Details
+Acceptance Criteria
+User is able to add the Skills
+
+@mytag
+ Scenario: 01 Test Add Skill sucessfully
+		Given User enter skill as "CPP"
+        And   select level as  "Beginner"
+		When  clicks on Add  button 
+		Then  skill should be added to your profile. The alert message, "CPP has been added to your skills" will be displayed.
+
+@mytag
+ Scenario: 02 Test Update skill sucessfully
+		Given User updates skill as "CPP"  
+		And   updates level as  "Expert"
+		When  clicks on Update  button 
+		Then  skill should be updated. The alert message, "CPP has been updated to your skills" will be displayed.
+
+@mytag
+ Scenario: 03 Test Delete skill sucessfully
+		When  User select to delete language  "CPP" 

```

Comment:
Missing Given keyword

Reply:
I don’t need we need to give the keyword ‘Given’ in all specflow tests. Given is actually used for specifying  a precondition. In all my specflow test cases my precondition is logging in the website which is already happening in setup() (start.cs). So I think I should start all my test cases with ‘When’ keyword. Please correct me if I am wrong